charts/drone/values.yaml

```diff
@@ -159,7 +159,7 @@ env:
   ## REQUIRED: Set the user-visible Drone hostname, sans protocol.
   ## Ref: https://docs.drone.io/installation/reference/drone-server-host/
   ##
-  DRONE_SERVER_HOST: ""
+  DRONE_SERVER_HOST: drone
```

Comment:
This should be the external server address (that you access in your browser) since it is used when creating the oauth2 redirect address and used to create the webhooks url (e.g. gitub webhook).  Not sure we would want this defaulted.

Reply:
> Not sure we would want this defaulted.

I changed it to `drone` because the current value of `""` was causing [a test to fail](https://cloud.drone.io/drone/charts/4/1/2).  We may want to handle that as a separate issue. I just fixed this opportunistically to make the test pass.

> This should be the external server address (that you access in your browser) since it is used when creating the oauth2 redirect address and used to create the webhooks url (e.g. gitub webhook).

I would argue that `DRONE_SERVER_PROXY_HOST` should be used for the Github webhook and the autocert.  The value of `DRONE_SERVER_PROXY_HOST` should default to `DRONE_SERVER_HOST` unless it is explicitly provided.

Reply:
> I would argue that DRONE_SERVER_PROXY_HOST should be used for the Github webhook

Yes this is how it currently works. These fields are optional and their intended purpose is when you are running Drone behind a firewall that cannot be publicly accessed, but you setup some sort of public endpoint to receive webhooks and forward them to your internal Drone instance.

Reply:
If the above file is being provided as an example values.yaml file, I recommend the following 
```
# DRONE_SERVER_PROXY_HOST: drone.external.domain.com 
DRONE_SERVER_HOST: drone.domain.local
```
Please let me know how you would like to proceed.

Reply:
Based on my experiences with metallb, ingress-nginx, and cert-manager, this values needs to be a value external domain name.  I submitted a change so the example value is `drone.example.com`.  Resolving, but please reach out if an alternative is recommended.